tests/test_game_time.py

```diff
@@ -60,3 +61,14 @@ def ow_player_data_response_not_found():
     ctx = await pg.prepare_game_times_context([OW_ID])
     result = await pg.get_game_time(OW_ID, ctx)
     assert result == GameTime(OW_ID, minutes, None)
+
+async def test_last_played_time(pg, config_data):
+    pg.local_client.config_parser = ConfigParser(config_data)
+
+    ctx = await pg.prepare_game_times_context(["21298"])
+    result = await pg.get_game_time("21298", ctx)
+    assert result == GameTime("21298", None, None)
+
+    ctx = await pg.prepare_game_times_context(["1214607983"])
```

Comment:
You can prepare just one context with both games - this is the way Galaxy does just before getting game_time for every game. For now does not matter as preparing context is always None, but this maybe changed in the future

```suggestion
    ctx = await pg.prepare_game_times_context(["21298", "1214607983"])
    result = await pg.get_game_time("21298", ctx)
    assert result == GameTime("21298", None, None)

```

Reply:
ok